src/core/packages/metrics/server-internal/src/metrics_service.test.ts

```diff
@@ -206,19 +205,19 @@ describe('MetricsService', () => {
 
       await expect(eluMetricsPromise).resolves.toEqual([
         expect.objectContaining({
-          short: expect.closeTo(0.1),
-          medium: expect.closeTo(0.1),
-          long: expect.closeTo(0.1),
+          short: expect.closeTo(0.0067, 3), // accumulating mean: (1.0 * 0.1) / 15 = 0.0067
+          medium: expect.closeTo(0.0033, 3), // accumulating mean: (1.0 * 0.1) / 30 = 0.0033
+          long: expect.closeTo(0.0017, 3), // accumulating mean: (1.0 * 0.1) / 60 = 0.0017
```

Comment:
```suggestion
          short: expect.closeTo(0.007, 3), // accumulating mean: (0 + 0 + 1) / (15 / 5) = 0.0067
          medium: expect.closeTo(0.003, 3), // accumulating mean: (0 + 0 + 1) / (30 / 5) = 0.0033
          long: expect.closeTo(0.002, 3), // accumulating mean: (0 + 0 + 1) / (60 / 5) = 0.0017
```

Reply:
[adjusts test input values to get predictable results](https://github.com/elastic/kibana/pull/236676/commits/d959d1db584999e30a6016afdf0192c2096f51df)